client/src/components/modal-forms/ManageGraduationForm.js

```diff
@@ -29,23 +30,54 @@ export const ManageGraduationForm = ({ initialValues, onClose, onSubmit }) => {
                   label='Has this participant graduated?'
                   options={[
                     {
-                      value: 'post_secondary_education_completed',
+                      value: postHireStatuses.postSecondaryEducationCompleted,
                       label: 'Graduated',
                     },
                     {
-                      value: 'failed_cohort',
-                      label: 'Failed',
+                      value: postHireStatuses.cohortUnsuccessful,
+                      label: 'Unsuccessful cohort',
                     },
                   ]}
                 />
                 {
-                  <DependantDateForm
-                    display={values.status === 'post_secondary_education_completed'}
+                  <Field
                     test-id={'editGraduationModalStatus'}
-                    name={'data.graduationDate'}
-                    label={'Graduation Date'}
+                    name={'data.date'}
+                    label={
+                      values.status === postHireStatuses.postSecondaryEducationCompleted
+                        ? 'Graduation Date'
+                        : 'Unsuccessful Graduation Date'
+                    }
+                    component={RenderDateField}
+                    maxDate={getTodayDate()}
                   />
                 }
+                {values.status === postHireStatuses.cohortUnsuccessful && (
+                  <Field
+                    test-id={'editGraduationModalRehire'}
```

Comment:
extremely minor but would be good if we can rename this from "rehire" to "unsuccessful" or something

Reply:
The value should be `continue`